### ISSUE TITLE
[Bug]: Video Z-Index Display Issue on Windows 10 using Intel Gen 11 CPUS

### ISSUE THREAD

**Issue by pitchinnate** [electron/electron#34829]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

19.0.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### What arch are you using?

x64

### Expected Behavior

When you have two div elements with videos stacked on top of each other and use css z-index to transition between them the div with the higher z-index should display on top.

```html
<div class="slideTop">
  <video src="video://./static/video1.mp4" playsinline loop muted ref="video" preload="auto" />
</div>
<div class="slideBottom" :class="{slideTopper: this.slideTopper}">
  <video src="video://./static/video2.mp4" playsinline loop muted ref="video2" preload="auto" />
</div>
```
```css
.slideTop {
    position: absolute;
    top: 50px;
    z-index: 10;
    width: 100%;
    height: 100%;
  }
  .slideBottom {
    top: 50px;
    position: absolute;
    z-index: 1;
    width: 100%;
    height: 100%;
  }
  .slideTopper {
    position: absolute;
    z-index: 20;
  }
```

### Actual Behavior

The div with the lower z-index displays on top. The CSS is applied correctly when looking at dev tools, but the div with the z-index of 10 shows up not the div with a z-index of 20.

![image](https://user-images.githubusercontent.com/2814435/177598350-4e0268bb-a518-4d13-892a-1f5f2662d3a9.png)

When the second div has only a z-index of 1 and the first div has a z-index of 10, the div with a z-index of 1 shows on top.

![image](https://user-images.githubusercontent.com/2814435/177598822-e8c88ede-49d9-4eb3-849b-417345b2d1b4.png)

### Additional Information

We have tested with multiple older generations of Intel CPUs and have no issues. Have also tried with multiple AMD CPUs without issues. We have also tried older versions of electron, back to 15 with the same results.

I have found a temporary fix:

By simply adding one more element (a 1px by 1px div with a mostly transparent background) on top with higher z-index than both videos, the z-index's on the divs with videos now work correctly.

A repo with an example of bug can be found at: https://github.com/pitchinnate/electron-video-demo

**Comment by RaisinTen** [comment]
> We have also tried older versions of electron, back to 15 with the same results.

Does that mean that the bug was introduced in v19 and v18 was the last known working electron version?

**Comment by pitchinnate** [comment]
> Does that mean that the bug was introduced in v19 and v18 was the last known working electron version?

No, this means that we don't know if there was ever a working version. The repo I shared uses electron version 19. I have tried with some old versions like version 15 and the bug existed there also. I haven't tried anything earlier than version 15 to know if it ever worked before then.

**Comment by pitchinnate** [comment]
We recently found a video encoding setting that does work correctly without having to use the temporary fix (1x1 pixel over top of videos). Figured I would post it in case it helps debug issue. When using Handbrake if you use the settings of h.264 10bit and adjust frame rate to 29.97 (NTSC Video) the videos seem to work correctly on the Intel Gen11 CPU and electron version 19.

![image](https://user-images.githubusercontent.com/2814435/179575453-75e61bf5-73b4-4ea0-8457-f48cdb4d32bf.png)
